### ISSUE TITLE
Temporary layer deactivation when key held

### ISSUE THREAD

**Issue by dickrp** [rvaiya/keyd#828]
Is there a way to have a toggle-activated layer temporarily deactivated for the duration of a key press? It is sometimes useful to toggle into a "task specific" layer that will be used for 95% of keypresses during a task. However, occasionally it is necessary to very briefly deactivate the "task specific" layer, i.e., revert to main. Deactivating the "task specific" layer during a key press would be less distracting than deactivating it and later reactivating it.

I don't see how to do this with toggle because it will permanently switch to the "normal layer" and also because main doesn't appear to be a proper layer to which one might switch.

In short, is there a way to get similar effect to an  unlayer() / unoverload() action?

**Comment by rvaiya** [comment]
Can you provide a sample use case?
